src/libraries/System.Private.CoreLib/src/System/HashCode.cs

```diff
@@ -319,6 +320,60 @@ public void AddBytes(ReadOnlySpan<byte> value)
             ref byte pos = ref MemoryMarshal.GetReference(value);
             ref byte end = ref Unsafe.Add(ref pos, value.Length);
 
+            if (Unsafe.ByteOffset(ref pos, ref end) < (sizeof(int) * 4))
+            {
+                goto Small;
```

Comment:
If value.Length is < 4, then this is going to jump to Small, which is positioned on a check to see if there's at least 4 bytes remaining.  Shouldn't there be an additional label on the one-byte-at-a-time loop and this would jump to that?

Reply:
Ugh, I misread the RHS. Nevermind.